core/services/ardupilot_manager/mavlink_proxy/test_all.py

```diff
@@ -82,34 +83,35 @@ def test_endpoint_validators() -> None:
         Endpoint.is_mavlink_endpoint({"connection_type": "potato", "place": serial_port_name, "argument": 100})
 
 
-def test_mavproxy(valid_output_endpoints: Set[Endpoint], valid_master_endpoints: Set[Endpoint]) -> None:
-    if not MAVProxy.is_ok():
-        warnings.warn("Failed to test mavproxy service", UserWarning)
-        return
-
-    assert AbstractRouter.get_interface("MAVProxy"), "Failed to find interface MAVProxy"
-
-    mavproxy = MAVProxy()
-    assert mavproxy.name() == "MAVProxy", "Name does not match."
-    assert re.search(r"\d+.\d+.\d+", str(mavproxy.version())) is not None, "Version does not follow pattern."
-
-    allowed_output_types = [
-        EndpointType.Serial,
-        EndpointType.UDPServer,
-        EndpointType.UDPClient,
-        EndpointType.TCPServer,
-        EndpointType.TCPClient,
-    ]
-    allowed_master_types = [
-        EndpointType.Serial,
-        EndpointType.UDPServer,
-        EndpointType.UDPClient,
-        EndpointType.TCPServer,
-        EndpointType.TCPClient,
-    ]
-    run_common_routing_tests(
-        mavproxy, allowed_output_types, allowed_master_types, valid_output_endpoints, valid_master_endpoints
-    )
+# TODO: MavProxy is failling to start for several endpoint combinations. Fix before reactivating its tests.
```

Comment:
You can just use a decorator over commenting all the code. 
`@pytest.mark.skip(reason="MavProxy tests are failling for several endpoint combinations. Since it's not being used and it's not a priority to support it, they are being temporarly disabled.")`

Reply:
Didn't know this one. Cool!